client/plots/sampleScatter.js

```diff
@@ -204,15 +204,17 @@ class Scatter {
 			chartType: 'sampleScatter',
 			settingsKey: 'minDotSize',
 			title: 'Minimum area, in square pixels',
-			min: 0
+			min: 9,
+			max: 3000
 		}
 		const maxDotSizeOption = {
 			label: 'Max area',
 			type: 'number',
 			chartType: 'sampleScatter',
 			settingsKey: 'maxDotSize',
 			title: 'Maximum area, in square pixels',
-			min: 0
+			min: 9,
```

Comment:
also the min. and max. should not be the same I think since this is a scaling. Do you think we should add some separate limits that can be reversed based on the scale order?

Reply:
I just saw your comments Karishma. The min and max are just a validation to avoid that nothing breaks. And I think now nothing breaks or looks out of place. Thanks for the feedback. I am happy to fix bugs but we cant make it perfect at once. I need to work on some other stuff. But feel free to improve it!